src/components/profile/VolunteeringHistory.jsx

```diff
@@ -3,8 +3,16 @@ import axios from 'axios';
 import { FieldTimeOutlined, ScheduleOutlined } from '@ant-design/icons';
 import { ConfigProvider, Table, Button } from 'antd';
 import './VolunteeringHistory.css';
-import EditHours from '../volunteer-profile-history/EditHours';
-import SuccessModal from '../volunteer-profile-history/SuccessModal';
+import styled from 'styled-components';
+import EditHours from '../../components/volunteer-profile-history/EditHours';
```

Comment:
🚫 **[eslint]** <[import/no-useless-path-segments](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-useless-path-segments.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Useless path segments for "../../components/volunteer-profile-history/EditHours", should be "../volunteer-profile-history/EditHours"
```suggestion
import EditHours from "../volunteer-profile-history/EditHours";
```